source/invoker.js

```diff
@@ -27,6 +27,16 @@ import toString from './toString';
  *      sliceFrom(6, 'abcdefghijklm'); //=> 'ghijklm'
  *      const sliceFrom6 = R.invoker(2, 'slice')(6);
  *      sliceFrom6(8, 'abcdefghijklm'); //=> 'gh'
+ *
+ *      const dog = {
+ *        speak: async () => 'Woof!'
+ *      };
+ *      const person = {
+ *        speak: async () => 'Hi!'
+ *      };
+ *      const speak = R.invoker(0, 'speak');
+ *      speak(dog).then(R.identity) //=> 'Woof!'
+ *      speak(person).then(R.identity) //=> 'Hi!'
```

Comment:
I like to reserve `//=>` for `returns`.  I think perhaps `//~>` would work here to imply that it logs that result.

Reply:
Fair enough will update!

Reply:
It actually doesn’t log that result and in fact would be the evaluates value. Looking at this again, I think its consistent with the above example and would argue it should remain as is. I don’t feel strongly though.

Reply:
Hm, how about
```js
speak(dog) //=> Promise.resolve('Woof!')
speak(person) //=> Promise.resolve('Hi!')
```

Reply:
IMO that’s not as clear as just doing the dance of “unwrapping” with identity. Perhaps I could open up a closure and put the comment in there?

Reply:
Invoker should obviously work with async functions,
perhaps the additional examples aren't necessary?

Reply:
Updated to use `console.log` and `~>`.